code/obj/item/gun/energy.dm

```diff
@@ -214,26 +216,185 @@ TYPEINFO(/obj/item/gun/energy)
 			projectiles += current_projectile
 
 
-////////////////////////////////////// Antique laser gun
-// Part of a mini-quest thing (see displaycase.dm). Typically, the gun's properties (cell, projectile)
-// won't be the default ones specified here, it's here to make it admin-spawnable (Convair880).
+//Antique laser gun
```

Comment:
All these defines should be undef'd at the end of the definition as they are not used elsewhere.